api/desecapi/exception_handlers.py

```diff
@@ -0,0 +1,32 @@
+from django.db.utils import OperationalError
+from django.utils import six
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import exception_handler, set_rollback
+
+
+def handle_db_unavailable(exc, context):
+    """
+    desecapi specific exception handling. If no special treatment is applied,
+    we default to restframework's exception handling. See also
+    https://www.django-rest-framework.org/api-guide/exceptions/#custom-exception-handling
+    """
+
+    if isinstance(exc, OperationalError):
+        if isinstance(exc.args, (list, dict, tuple)) and exc.args and \
+            exc.args[0] in (
+                2002,  # Connection refused (Socket)
+                2003,  # Connection refused (TCP)
+                2005,  # Unresolved host name
+                2007,  # Server protocol mismatch
+                2009,  # Wrong host info
+                2026,  # SSL connection error
+        ):
+            # Gracefully let clients know that we cannot connect to the database
+            msg = _('Please try again later.')
```

Comment:
Why do we use i18n here while we use it nowhere else?

Reply:
This follows the implementation that's also [used in rest framework 3.7](https://github.com/encode/django-rest-framework/blob/3.7.7/rest_framework/views.py#L63), however was changed in 3.8 and 3.9 (current). That is, our code currently follows 3.7 style, which is in line with our `requirements.txt` (as upgrading is blocked by the bulk restframework module).

Which style should we follow?

Reply:
As discussed offline, we will stick to restframework 3.7 style, but remove the i18n support.